partials/tabs.php

```diff
@@ -0,0 +1,181 @@
+			}
+			$tabsize = strlen( $tabtitle ); // start a count of characters in the tabs. If we get too silly with size, revert to contents display
+            $tabsize += 12;
+			$menu          = $link;
+			$altmenu = $altlink;
+
+			// Get all child/sibling pages (depending on whether this is a parent or child page) that use this tabbed page template
+			$args     = array(
+				'post_type'   => 'page',
+				'post_parent' => $post_parent,
+				'orderby'     => 'menu_order',
+				'order'       => 'ASC',
+				'depth'       => 1,
+				'post_status' => 'publish',
+				'meta_query'  => array(
+					array(
+						'key'   => 'tabbed-page',
+						'value' => 'on',
+					),
+				),
+			);
+			$children = new WP_Query( $args );
+
+			// Define tab icons (based on image sprite definitions in Nightingale)
+			$icons = array( "user", "menu", "info", "pencil", "currency", "speaker" );
+			// Display child/sibling tabs
+			while ( $children->have_posts() ) {
+				$children->the_post();
+				// Build tab
+				$link = '<li class="nhsuk-bordered-tabs-item ';
+				$altlink = '<li class="nhsuk-contents-list__item" ';
+				$tabname = get_post_meta( $post->ID, 'tabname', true );
+				if ( $tabname ) {
+					$tabtitle = $tabname;
+				} else {
+					$tabtitle = get_the_title( $post->ID );
+				}
+				if ( is_page( $post->ID ) ) {
+					$link .= 'nhsuk-bordered-tabs-item-active';
+					$link .= '">' . $tabtitle . '</li>';
+					$altlink .= 'aria-current="page"><span class="nhsuk-contents-list__current">' . $tabtitle . '</span></li>';
+				} else {
+					$link .= '"><a class="nhsuk-bordered-tabs-link';
+					$link .= '" href="';
+					$link .= get_permalink( $post );
+					$link .= '">';
+					$link .= $tabtitle;
+					$link .= '</a></li>';
+					$altlink .= '><a class="nhsuk-contents-list__link" href="' . get_permalink( $post ) . '">' . $tabtitle . '</a></li>';
+				}
+				$tabsize += strlen( $tabtitle );
+				$tabsize += 12;
+				// If number of tabs exceeds number of icons, reset to start of icon array (icons will repeat from the start)
+				if ( ! next( $icons ) ) {
+					reset( $icons );
+				}
+				// Display tab
+				$menu .= $link;
+				$altmenu .= $altlink;
+			}
+			wp_reset_query();
+
+		}
+		?>
+
+        <div class="nhsuk-full-width-container">
+            <section class="nhsuk-hero" id="nhsuk-tabbed-title">
+
+                <div class="nhsuk-width-container">
+                    <div class="nhsuk-grid-row">
+                        <div class="nhsuk-grid-column-two-thirds">
+                            <div class="nhsuk-hero__wrapper--tabs">
+                                <h1 class="nhsuk-u-margin-bottom-3 nhsuk-heading-l entry_title"><?php echo $section_title; ?></h1>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </section>
+        </div>
+		<?php echo nightingale_breadcrumb(); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
```

Comment:
There is a security concern here, do we display unescaped content?

Reply:
yes. It is already pre-processed and sanitised in the function.